lib/mrb/scripts/table.rb

```diff
@@ -102,7 +102,7 @@ def ensure_sort_key(key, converted_keys)
         order    = key[:order] || order
         offset   = key[:offset] || offset
       else
-        key_name = key
+        key_name = key.dup
```

Comment:
We should not use `#dup`. We should stop to update `key_name` directory by `key_name[0] = ""` instead. `key_name = key_name[1..-1]` is alternative way.